### DISCUSSION TITLE
Достижения

### DISCUSSION THREAD

**Discussion by MapkaN1** [LacunaHub/.github#20] [Предложения]
Столкнулся на днях с таким вот ботом достижений информацию о котором наотрез отказались давать админы сервера. Долго искал и не нашёл чего то подобного. Возможно ли что в будущем у лакуны будет подобные достижения?? Если да то это будет шедеврально!!!
![Uploading Screenshot_20240512_231633_com.discord.jpg…]()

**Comment by rovius** [comment]
Непонятно, о каких достижениях идёт речь. Скриншот не загрузился. К тому же, за два года никакой активности по этому предложению. Закрываю.
